static/js/marker.js

```diff
@@ -127,8 +127,9 @@ var MarkerView = Backbone.View.extend({
 
     },
     unhighlight : function() {
-    	if (app.oms.markersNearMarker(this.marker, true)[0]){
-    		this.marker.setIcon(this.getIcon("multiple", SEVERITY_VARIOUS));
+    	if (app.oms.markersNearMarker(this.marker, true)[0] && !this.model.get("currentlySpiderfied")){
+            var group = this.model.get("groupID") -1;
```

Comment:
Why subtract 1? Maybe group IDs should just start from 0 and then they're exactly the index in the array.